[[Bug 11796]] iPhoneFileDataProtection empty result

seaniepie opened the pull request:
- Added tFile to the function in the example
- Added description to all results
- Added empty to the list of results

seaniepie committed:
[[Bug 11796]] iPhoneFileDataProtection empty result

- Added tFile to the function in the example
- Added description to all results
- Added empty to the list of results

livecodeali commented on docs/dictionary/function/iphoneFileDataProtection.lcdoc:
```diff
@@ -24,10 +24,11 @@ A string containing the path to the file.
+-   complete: The file is not accessible, for read or write, while the device is locked.
+-   complete unless open: The file is fully protected when the device is locked, unless it was already open.
+-   complete until first user authentication: The file is fully protected until the user unlocks the device for this first time.
 
```
The values here (other than `empty`) need to be in double-quotes.

peter-b committed:
docs: Put enum values in quotes

Put the possible enumerated values returned by `iphoneFileDataProtection()` in quotes.

livecodeali commented:
@livecode-vulcan review ok 7cdefa0

livecode-vulcan commented:
💙 review by @livecodeali ok 7cdefa0610eb73e6d62d0e9fbcde7e3e44aae8f9

livecode-vulcan commented:
😎 test success 7cdefa0610eb73e6d62d0e9fbcde7e3e44aae8f9
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/476)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/481)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/476)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/476)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/482)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/476)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/476)
